docs/changelog.md

```diff
@@ -37,6 +37,68 @@ It doesn't include
 ## Unreleased
 
 
+### Highlights for users
+
+* Initial support for muted users. (PRs #4774, #4730, #4710, #4736, #4711,
+  #4699, #4678)
+* Added a setting to choose whether external links should open in the
+  embedded browser or the external browser. (PR #4679)
+* Added a server-compatibility notice for self-hosted orgs running Zulip
+  Server versions before 2.0.0. (PR #4750)
+* Stream settings screen now has a "subscribe" button. And, when
+  unsubscribed, useless toggle switches ("Pinned", "Muted", and
+  "Notifications") are hidden. (#4715)
+* Updated the primary app color to match the new logo. (PR #4544)
+* Fixed a bug on Android where the app sometimes opened to an unexpected
+  narrow when opening the app from the background. (#4758)
+* Fixed a glitch where the org URL input screen would flash by before the
+  authentication screen, on some app startups or when the user tapped a
+  logged-out account in the account picker. (#4604)
+* Fixed misconfigurations causing the Play Store to not offer the Android
+  app on devices that don't have cameras (#4722) or WiFi capability (PR
+  #4761).
+* Continued work toward enabling Android share-to-Zulip. (#4514)
+* Unclear, little-used "create group PM" option in the compose box removed.
+  (PR #4777)
+* Miscellaneous UI improvements. (#4734; PRs #4717, #4749, #4716, #4697,
+  #4686, #3517, #3395)
+
+Plus, like every release, numerous other fixes and improvements for your
+Zulip experience.
+
+
+### Highlights for developers
+
+* Extended Jest coverage to include Android-only codepaths. (#4700)
+* All `yarn` dependencies upgraded as far as possible under semver
+  constraints, as of 2021-06-07. (PR #4789)
+* Fixed new iOS build failures. (PRs #4721, #4634)
+* `restart` events can now cause updates to `zulipVersion` and
+  `zulipFeatureLevel` in `state.accounts[0]`. (PR #4707)
+* Increased test coverage of our storage logic, and continued cleaning up
+  `redux-persist`. (#4709)
+* Miscellaneous testing improvements. (PRs #4760, #4753)
+* Fixed a bug where `CaughtUp` state was being wrongly overwritten. (PR
+  #4698)
+* In dev mode only, a user-facing string passed to a `_: GetText` will now
+  be highlighted in the UI if it doesn't have an entry in
+  `messages_en.json`. (#4728)
+* Cleaned up some code for particular API endpoints. ( PR #3955)
+* Bumped `@zulip/shared` to v0.0.5. (PR #4680)
+* Added a migration that should have been released with v27.162. (PR #4689)
```

Comment:
This sounds kind of alarming. Could use another word or two to make clear that the impact of the missing migration was harmless.

(Alternatively, this item doesn't super need to be here at all -- it's not something that even developers especially need to know happened.)